packages/app-webdir-ui/src/helpers/dataConverter.js

```diff
@@ -220,20 +220,20 @@ export const staffConverter = (
   // We guard against null asurite_id being returned from data source in some
   // instances by using a conditional render.
```

Comment:
Need to update the comment to match the changes.

And the student converter needs to be updated to match this logic as well.